src/generate-manifest.js

```diff
@@ -58,17 +59,62 @@ const generateManifest = async function (codiusVarsPath, codiusPath) {
   }
 
   removeDescriptions(generatedManifest) // remove description fields from manifest
-  debug(`Generated Manifest: ${JSON.stringify(generatedManifest, null, 2)}`)
+
+  // check the digest of each container image
+  const containers = generatedManifest['manifest']['containers']
```

Comment:
this variable isn't being used; can you run lint

Reply:
The containers variable is being used in the for loop in the next line